docs/fsharp/language-reference/pattern-matching.md

```diff
@@ -37,11 +37,11 @@ Supported patterns are shown in the following table. At run time, the input is t
 |Identifier pattern|A case value of a discriminated union, an exception label, or an active pattern case|`Some(x)`<br /><br />`Failure(msg)`|
 |Variable pattern|*identifier*|`a`|
 |`as` pattern|*pattern* as *identifier*|`(a, b) as tuple1`|
-|OR pattern|*pattern1* &#124; *pattern2*|`([h] &#124; [h; _])`|
-|AND pattern|*pattern1* &amp; *pattern2*|`(a, b) &amp; (_, "test")`|
+|OR pattern|*pattern1* &#124; *pattern2*|<code>([h] &#124; [h; _])</code>|
+|AND pattern|*pattern1* &amp; *pattern2*|`(a, b) & (_, "test")`|
 |Cons pattern|*identifier* :: *list-identifier*|`h :: t`|
 |List pattern|[ *pattern_1*; ... ; *pattern_n* ]|`[ a; b; c ]`|
-|Array pattern|[&#124; *pattern_1*; ..; *pattern_n* ]|`[&#124; a; b; c &#124;]`|
+|Array pattern|[&#124; *pattern_1*; ..; *pattern_n* ]|<code>[&#124; a; b; c &#124;]</code>|
```

Comment:
Looks like there's a `&#124;` missing at the end of `*pattern_n*` in the second column of this line.  I think it was missed in the initial move to docs.microsoft.com.  Could you insert that?  I'll merge after that, since your previous changes look good on our staging site 😄:

<img width="178" alt="screen shot 2016-08-30 at 7 05 49 am" src="https://cloud.githubusercontent.com/assets/6309070/18092202/4874bcaa-6e80-11e6-9f81-539b4c52bc88.png">